src/TurboSMSMessage.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace NotificationChannels\TurboSMS;
+
+class TurboSMSMessage
+{
+    public $to;
```

Comment:
this doesn't seem to be used?

Reply:
It is used here https://github.com/laravel-notification-channels/channels/pull/83/files#diff-144fe7fe2a63a5b4085e8375b633c683R101

Reply:
oh, I see. I don't think it should be on the message class, because it's just assigned and read back. In your link, the var could just be name `$to` and it would have the same behaviour.

If someone tried to set the `$message->to` to something, it would just be overridden, so i'm not sure what benefit having it as a property provides

Reply:
I agree. I just tried to keep class structure style of other channels in this repo. To be consistent we should remove the message class itself, because the "body" property also can be replaced by local var.
What do you think about it?

Reply:
I think we should keep the message class, since in the future it might be needed to add additional options to it :)